azure-pipelines.yml

```diff
@@ -11,73 +11,41 @@ steps:
 - checkout: self
   clean: true
 - powershell: |
-    $line = '=' * 80
-    Write-Host "$line`nWindows`n$line`n"
-    Get-ComputerInfo |
-      select WindowsProductName, WindowsVersion, OsHardwareAbstractionLayer |
-      Out-String |
-      Write-Host
-    #
-    # Azure
-    #
-    $assetTag = Get-WmiObject -class Win32_SystemEnclosure -namespace root\CIMV2 |
-      Select -ExpandProperty SMBIOSAssetTag
-
-    # only Azure VMs have this hard-coded DMI value
-    if ($assetTag -eq '7783-7084-3265-9085-8269-3286-77')
-    {
-      Write-Host "`n`n$line`nAzure`n$line`n"
-      Invoke-RestMethod -Headers @{'Metadata'='true'} -URI http://169.254.169.254/metadata/instance?api-version=2017-12-01 -Method Get |
-        ConvertTo-Json -Depth 10 |
-        Write-Host
-    }
-    #
-    # Docker
-    #
-    Write-Host "`n`n$line`nDocker`n$line`n"
-    docker version
-    docker images
-    docker info
-    sc.exe qc docker
-    #
-    # Ruby
-    #
-    Write-Host "`n`n$line`nRuby`n$line`n"
-    ruby --version
-    gem --version
-    gem env
-    bundle --version
-    #
-    # Environment
-    #
-    Write-Host "`n`n$line`nEnvironment`n$line`n"
-    Get-ChildItem Env: | % { Write-Host "$($_.Key): $($_.Value)"  }
+    $gemfile = Join-Path -Path (Get-Location) -ChildPath 'docker/Gemfile'
```

Comment:
Usually something like `Get-Location` is unreliable. which is why I have that `Get-CurrentDirectory` helper. I also think you can just use `.` there, like:

```powershell
$gemfile = Join-Path -Path '.' -ChildPath 'docker/Gemfile'
```

Reply:
I didn't want to copy/paste that custom `Get-CurrentDirectory` function into every repo, and it seemed like I should be able to use the PowerShell built-in function `Get-Location`.

I also tried `$PWD` and that worked; I feel like I had a problem trying to use `.`.

Reply:
Happy to rewrite these, I just don't want to use a copy/pasted custom function if we can avoid it.